### PR TITLE
mgr/dashbaord: Fix bogus type annotations

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tspmelo** [comment]
There is a typo on the commit and PR title: "mgr/dashb**ao**rd:"

File: src/pybind/mgr/dashboard/controllers/prometheus.py [hunk 1]
```diff
@@ -27,20 +33,21 @@ def fetch_alert(self, **notification):
...
     def prometheus_proxy(self, method, path, params=None, payload=None):
-        # type (str, str, dict, dict)
+        # type: (str, str, dict, dict) -> Any
         return self._proxy(self._get_api_url(Settings.PROMETHEUS_API_HOST),
                            method, path, 'Prometheus', params, payload)
 
     def alert_proxy(self, method, path, params=None, payload=None):
-        # type (str, str, dict, dict)
+        # type: (str, str, dict, dict) -> Any
+        i:int = "s"
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard/controllers/prometheus.py:42]: whoops. this needs to be removed

File: src/pybind/mgr/dashboard/controllers/prometheus.py [hunk 2]
```diff
@@ -27,20 +33,21 @@ def fetch_alert(self, **notification):
 
 class PrometheusRESTController(RESTController):
     def prometheus_proxy(self, method, path, params=None, payload=None):
-        # type (str, str, dict, dict)
+        # type: (str, str, dict, dict) -> Any
```
Comment (votdev) [inline src/pybind/mgr/dashboard/controllers/prometheus.py:36]: To me it looks like `Dict` is returned by `self._proxy`.

**Comment by sebastian-philipp** [comment]
no bandwith to do that right now.

**Pull request closed without merging**